frontend/app/component/form/polling_station_choice/PollingStationChoiceForm.test.tsx

```diff
@@ -22,13 +23,29 @@ describe("Test PollingStationChoiceForm", () => {
     await user.type(pollingStation, "1234567");
     expect(pollingStation).toHaveValue("123456");
 
-    expect(screen.getByText("Kies het stembureau")).not.toBeVisible();
+    await user.click(submitButton);
+  });
+
+  test("Polling station list", async () => {
```

Comment:
I'd prefer an `overrideOnce()` at the start of this test for the api response instead of relying on what's in `frontend/lib/api-mocks/PollingStationMockData.ts`.

In this case the mock plainly returns the data, so it's unlikely to lead to isses. For the `data_entries` ednpoint it does not. So I'd rather have the rule "always do an `overrideOnce()`", which leaves no doubt what data is used by test, than the rule "it depends on how complicated someone judged the mock implementation to be".

Reply:
I opened https://github.com/kiesraad/abacus/issues/163 to address this